napari/components/layerlist.py

```diff
@@ -27,6 +27,14 @@ def __init__(self, data=()):
             basetype=Layer,
             lookup={str: lambda e: e.name},
         )
+        self.selection.events.connect(self._on_selection_event)
+
+    def _on_selection_event(self, event):
```

Comment:
Will we always need this or is this temporary for backwards compatibility? Would be nice if we didn't need